google/resource_container_cluster.go

```diff
@@ -172,6 +172,36 @@ func resourceContainerCluster() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{"logging.googleapis.com", "none"}, false),
 			},
 
+			"maintenance_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"daily_maintenance_window": {
+							Type:     schema.TypeList,
+							Required: true,
```

Comment:
Just for the sake of future-proofing, let's go ahead and `ForceNew: true` this one too